openlibrary/templates/about/team.json

```
@@ -0,0 +1,142 @@
+[
+	{
+		"name": "Drini Cami",
+		"status": "Staff",
+		"role": "",
+		"bio": "",
+		"photo_url": "https://scontent-lga3-2.xx.fbcdn.net/v/t39.30808-1/347564285_1478873239584126_8295394847869589900_n.jpg?stp=dst-jpg_p320x320&_nc_cat=107&ccb=1-7&_nc_sid=7206a8&_nc_ohc=pXhxdwYi9r4AX_j6UcQ&_nc_ht=scontent-lga3-2.xx&oh=00_AfAbI_BP0B9px9S1H68Y6Kxhhf8HRpIpQtRq2fHhvyBjAw&oe=6483BD02",
+		"ol_user_url": "https://openlibrary.org/people/ScarTissue",
+		"personal_url": "",
+		"favorite_book_url": ""
+	},
+	{
+		"name": "Jim Champ",
+		"status": "Staff",
+		"role": "Developer",
+		"bio": "",
+		"photo_url": "https://avatars.githubusercontent.com/u/28732543?v=4",
+		"ol_user_url": "https://openlibrary.org/people/jchamp_archive",
+		"personal_url": "",
+		"favorite_book_url": ""
+	},
+	{
+		"name": "Mek",
+		"status": "Staff",
+		"role": "Chief of Staff",
+		"bio": "Over 70% of the world lives on less than $10 a day. This means up to 2/3 of our neighbors can't afford all the resources they need to reach their full potential. Because of this, we're letting billions of...",
+		"photo_url": "https://cyber.harvard.edu/sites/default/files/styles/teaser/public/2022-08/berkman-photo.png?h=6cb7e18c&itok=rXzz8t8C",
+		"ol_user_url": "https://openlibrary.org/people/mekBot",
+		"personal_url": "https://mek.fyi/",
+		"favorite_book_url": ""
+	},
+	{
+		"name": "Scott Barnes",
+		"status": "Staff",
+		"role": "Programmer",
```

```suggestion
		"role": "Developer",
```
This one is on me. :) We should probably use the same names for the same types of positions. I will update the spreadsheet to reflect this change.

---

Changed!